Frontend/CO.CDP.OrganisationApp/Pages/Organisation/OrganisationPponSearch.cshtml

```diff
@@ -0,0 +1,167 @@
+                <div class="govuk-form-group">
+                    <label class="govuk-label govuk-!-font-weight-bold" for="sort-order">
+                        @StaticTextResource.Global_Sort_Results
+                    </label>
+                    <select class="govuk-select" id="sort-order" name="sortOrder" aria-controls="sort-submit">
+                        <option value="rel"
+                                selected="@(Request.Query["sortOrder"] == "rel" ? "selected" : null)">
+                            @StaticTextResource.Global_Sort_Relevance
+                        </option>
+                        <option value="asc"
+                                selected="@(Request.Query["sortOrder"] == "asc" ? "selected" : null)">
+                            @StaticTextResource.Global_Sort_Alphabetical_Asc
+                        </option>
+                        <option value="desc"
+                                selected="@(Request.Query["sortOrder"] == "desc" ? "selected" : null)">
+                            @StaticTextResource.Global_Sort_Alphabetical_Desc
+                        </option>
+                    </select>
+                    <button type="submit" id="sort-submit" class="govuk-visually-hidden">
+                        @StaticTextResource.Global_Sort_Txt
+                    </button>
+                    <noscript>
+                        <button type="submit" class="govuk-button govuk-!-margin-left-2">@StaticTextResource.Global_Sort_Txt</button>
+                    </noscript>
+
+                </div>
+            </div>
+            }
+        </div>
+    }
+</form>
+
+@if (Model.TotalOrganisations > 0)
+{
+    foreach (var org in Model.Organisations)
+    {
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <h2 class="govuk-heading-m">@org.Name</h2>
+                @if (org.Identifiers.Any(i => i.Scheme.Equals("GB-PPON") && i.Id != null))
+                {
+                    <p class="govuk-body govuk-!-margin-bottom-1">
+                        <span class="govuk-!-font-weight-bold">PPON</span>
+                        @org.Identifiers.First(i => i.Scheme.Equals("GB-PPON")).Id
+                    </p>
+                }
+                <p class="govuk-body">
+                    <span class="govuk-!-font-weight-bold">Organisation address</span>&nbsp;
+                    @Model.FormatAddresses(org.Addresses)
+                </p>
+            </div>
+            <div class="govuk-grid-column-one-third govuk-!-text-align-right">
+                <div class="govuk-grid-row" style="text-align: right;margin-right:0">
+                    @foreach (var partyRole in @org.Roles)
+                    {
+                        <strong class="govuk-tag @Model.GetTagClassForRole(partyRole)">
+                            @partyRole
+                        </strong>
```

Comment:
Use party role display extension to handle the possibility of non-distinct roles as we discussed

Reply:
This is fine - but it'll need to be changed in future for Welsh